DataDictionary.md

```diff
@@ -33,6 +33,7 @@
 3. Criminal Actions - Severe action reserved for facilities intentionally committed
 
 ## Penalty
+* Penalties most often incur a fee as a result of noncompliance with enviromental regulations
 
 ## Compliance, Noncompliance, and Significant Noncompliance
 * Compliance represents if a facility adheres to environmental regulations
```

Comment:
Slightly awkward phrasing, maybe "A facility is said to be in compliance if it adheres to environmental regulations"